django_celery_beat/models.py

```diff
@@ -45,6 +45,17 @@ def cronexp(field):
     """Representation of cron expression."""
     return field and str(field).replace(' ', '') or '*'
 
+def crontab_schedule_celery_timezone():
```

Comment:
can you test this with unit test?

Reply:
Test added. Also, I've fixed the double space on `help_text` parameter of `timezone` `CrontabSchedule` model field taking advantage of the new migration.

Reply:
A hardcode CELERY_TIMEZONE still here, when I `app.config_from_object('django.conf:settings', namespace='CELERY_OTHER')`, the setting name will be `CELERY_OTHER_TIMEZONE` but not `CELERY_TIMEZONE`.

Reply:
can you comment on Belongwqz comment

Reply:
Okay, seems that this do the trick:

```python
from celery import current_app

CELERY_TIMEZONE = getattr(settings, '%s_TIMEZONE' % current_app.namespace)
```

Can you review it again?

Reply:
Yes, perfect!